x-pack/plugins/canvas/canvas_plugin_src/functions/common/__tests__/axis_config.js

```diff
@@ -50,9 +50,9 @@ describe('axisConfig', () => {
         expect(result).to.have.property('position', 'bottom');
       });
 
-      it('defaults to an empty string if not provided', () => {
+      it('defaults to "left" if not provided', () => {
```

Comment:
Curious, was this test not running before? Seems like it would have always failed, the default isn't changed in this PR...